### ISSUE TITLE
step-70: included prms are broken

### ISSUE THREAD

**Issue by tjhei** [dealii/dealii#10273]
The included prms ``parameters.prm`` and ``parameters_23.prm`` are not descriptive (what is the difference?) and don't work. Annoyingly, they crash with the indescriptive error
```
Running StokesImmersedProblem<2,3> using Trilinos.
---------------------------------------------------------
TimerOutput objects finalize timed values printed to the
screen by communicating over MPI in their destructors.
Since an exception is currently uncaught, this
synchronization (and subsequent output) will be skipped
to avoid a possible deadlock.
---------------------------------------------------------


----------------------------------------------------
Exception on processing: 

--------------------------------------------------------
An error occurred in line <1360> of file </ssd/deal-git/source/base/parameter_handler.cc> in function
    void dealii::ParameterHandler::print_parameters(const std::string &, const ParameterHandler::OutputStyle) const
The violated condition was: 
    out
Additional information: 
    An input/output error has occurred. There are a number of reasons why this may be happening, both for reading and writing operations.
```

@luca-heltai ?

**Comment by luca-heltai** [comment]
The tutorial explains what is the difference between the two. I think the issue is in the `output directory `results_23` which is non-existant. We should set that parameter to `.`.

**Comment by luca-heltai** [comment]
The same is true for the `parameters.prm`. We should either add the two directories, or change the value of the parameter to `.`. I think it was a mistake to add the files. The program generates working ones by default.

**Comment by luca-heltai** [comment]
How do you want me to proceed?

**Comment by tjhei** [comment]
> How do you want me to proceed?

The most simple fix is fine for 9.2. Longer term, I would prefer the code to just auto-create the directory.
